fhir-server/src/main/java/com/ibm/fhir/server/util/RestAuditLogger.java

```diff
@@ -463,11 +463,18 @@ private static void logBundleBatch(AuditLogService auditLogSvc, HttpServletReque
                     entry.getContext().setAction(action);
                 }
 
+                String loc = null;
+                if (responseEntry.getResponse() != null && responseEntry.getResponse().getLocation() != null) {
+                    loc = responseEntry.getResponse().getLocation().getValue();
+                } else { 
+                    loc = requestEntry.getRequest().getUrl().getValue();
```

Comment:
I wonder if we should just always use the requestEntry url...
any reason you chose the response URL (via response.location) instead?

Reply:
I think I saw that it was populated.  Are there better more reliable locations? 
And yeah we could use the requestEntry's url in this case since it's batch.

Reply:
to me it boils down to whether we're logging the request url or the url that is the target of the operation.
typically those would be the same for "normal" updates and deletes.

however, for create I think they would differ.
* request: `[base]/[resourceType]`
* response: `[base]/[resourceType]/_history/1`

Reply:
I believe they could also differ for conditional create / conditional update cases

Reply:
Yes, they will differ from what I've seen. - let's use the request in this case.

Reply:
Update is pushed, I'll leave it to you to merge (if ok)